src/scratch_impl.h

```diff
@@ -60,6 +60,10 @@ static size_t secp256k1_scratch_max_allocation(const secp256k1_callback* error_c
         secp256k1_callback_call(error_callback, "invalid scratch space");
         return 0;
     }
+    /* Ensure that multiplication will not wrap around */
+    if (objects > SIZE_MAX/(ALIGNMENT - 1)) {
```

Comment:
`ALIGNMENT == 1` should be handled.

Reply:
I don't think that's necessary but I can fix it. Any suggestion for how to handle it? Is a VERIFY_CHECK enough?

Reply:
Yeah, I don't know either how big of  a problem it is. But it's not impossible. I can imagine that someone defines  `ALIGNMENT == 1` because, e.g., on x86, unaligned access is not an issue.

I think we should either
  * `#error` out at compile-time (better than with at runtime `VERIFY_CHECK`)
  * or just handle it. An `if (ALIGNMENT == 1)` can anyway be optimized out then by the compiler.

Reply:
Added commit to handle that at runtime now.